commands/dkcheck.ml

```diff
@@ -124,6 +130,15 @@ let _ =
     ; ( "-q"
       , Arg.Unit (fun () -> Debug.set_debug_mode "q")
       , " Quiet mode (equivalent to -d 'q'" )
+    ; ("-sz"
+      , Arg.Set sizechange
+      , "Apply Size Change Principle" )
+    ; ("-szv"
+      , Arg.Set szvb
+      , "Apply Size Change Principle and verbose it" )
+    ; ("-szst"
+      , Arg.Set szstat
+      , "Apply Size Change Principle and print results useful for stats" )
```

Comment:
It is not very nice to have all these specific options. You should use a general verbose option, possibly with some integer level.

Reply:
I agree it is quite agly, but when I debug my code, I like to activate the verbose only for the sizechange module.